sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BatchLoads.java

```diff
@@ -359,9 +363,15 @@ public String apply(Void input) {
                 new DoFn<String, String>() {
                   @ProcessElement
                   public void getTempFilePrefix(ProcessContext c) {
+                    String tempLocationRoot;
+                    if (customGcsTempLocation != null && customGcsTempLocation.isAccessible()) {
```

Comment:
Inside a DoFn it must always be accessible: if it's inaccessible, that's a bug, rather than a situation requiring fallback to line 370.

Reply:
done